validators/http/http.go

```diff
@@ -105,6 +108,31 @@ func lookupStatusCode(statusCode int, statusCodeRanges []*numRange) bool {
 	return false
 }
 
+// lookupHTTPHeader looks up for header and value in HTTP response.
+// returns true on first match. If value_regex is omitted - check for header existense only
+func lookupHTTPHeader(headers nethttp.Header, expectedHeader string, valueRegexp *regexp.Regexp) bool {
+
+	values, ok := headers[expectedHeader]
+
+	if !ok {
+		return false
+	}
+
+	if ok && valueRegexp == nil {
```

Comment:
We don't need to check for "ok" again. We'll come here only if header is found.

Reply:
good catch